src/en/components/buttons/buttons.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Button Component 
+layout: "layouts/base.njk"
+eleventyNavigation:
+  key: buttons
+  title: Button Component 
+  locale: en
+---
+
+# Button Component
+
+A button is an object to activate a response or a change of location or state. 
+
+## Function 
+
+Buttons highlight important actions that help a user achieve tasks.
+A button is initialized through a click or keydown action, such applying the Enter key when the button’s selected. The user communicates a response, changes their location, or changes their state by using the click or keydown action. 
+
+## What buttons are for
+
+Use buttons for important actions, such as:
+- saving, deleting, downloading 
+- clearing entered data 
+- submitting a form or request
+- giving consent or agreement
+- making a binary choice
+- entering or exiting a web product, like a secure area login, a form, or tutorial
+
+In applications, buttons are used in place of text links for things like:   
+- controlling movement, going from one screen to another on a website
+- external links  
+
+## What buttons are not for
+
+Consider links instead when:
+- taking a user off a website page.
+- an action is less important.
+
+Consider menus or selections, like dropdowns or radio buttons, instead when:
+- The user must choose between two or more options.
+
+[Implementation for Developing]({{ "/en/components/buttons/development" }}) | [Implementation for Designing]({{ "/en/components/buttons/design" }})
```

Comment:
Sorry this is my bad for not documenting this. For urls at the moment, they will need to be have the url filter on them to work with Github pages. This shouldn't be an issue if we switch to a domain but for now each link created will need to be in this format: `[Link text]({{ "url" | url }})`.

Again, my bad for not documenting this.

Reply:
Forgot to push this up earlier but I went back and reformatted those URLs

Reply:
My bad again, I should have used an actual link for the example.

The format is `[Implementation for Developing]({{ "/en/components/buttons/development" | url }})`